solidstart-tanstackquery-tailwind-modules/src/routes/[login].tsx

```diff
@@ -51,7 +51,7 @@ const Profile = () => {
   );
 
   const queryRepos = createQuery(
-    () => ['query-repos'],
+    () => [`query-repos-${location.query?.after}_${location.query?.before}`],
```

Comment:
This is interesting, why did you choose to do it this way?

Reply:
So this serves as a query key for TanStack, when the key changes it fires another API call.

Reply:
I guess what I meant was, was there a reason to do this as a string interpolation rather than something like:
```ts
createQuery(
    () => ['query-repos', { after: location.query?.after, before: location.query?.before }],
```

Reply:
No apparent reasons, this is also valid.